doc/api/timers.md

```diff
@@ -2,87 +2,128 @@
 Node.js makes no guarantees about the exact timing of when callbacks will fire,
 nor of their ordering. The callback will be called as close as possible to the
 time specified.
 
-To follow browser behavior, when using delays larger than 2147483647
-milliseconds (approximately 25 days) or less than 1, the timeout is executed
-immediately, as if the `delay` was set to 1.
+When `delay` is larger than `2147483647` or less than `1`, the `delay` will be
+set to `1`.
+
+If `callback` is not a function, an [`Error`][] will be thrown.
+
+## Cancelling Timers
+
+The `setImmediate()`, `setInterval()`, and `setTimeout()` methods each return
+opaque objects that represent the scheduled timers. These can be used to
+cancel the timer and prevent it from triggering.
+
+### clearImmediate(immediateObject)
+
+* `immediateObject` {Object} An immediate timer object as returned by
+  `setImmediate()`.
+
+Cancels an "immediate timer".
+
+### clearInterval(intervalObject)
+
+* `intervalObject` {Object} An interval timer object as returned by
+  `setInterval()`.
+
+Cancels an "interval timer".
+
+### clearTimeout(timeoutObject)
+
+* `timeoutObject` {Object} A timeout timer object as returned by `setTimeout()`.
+
+Cancels a "timeout timer".
+
+## Timer Reference Tracking
+
+By default, when a timer is scheduled using either `setTimeout()` or
+`setInterval()`, the Node.js event loop will continue running as long as the
+timer is active. Each of the opaque timer objects returned by these functions
+export both `timer.ref()` and `timer.unref()` functions that can be used to
+control this default behavior.
+
+### timer.ref()
+
+When called, requests that the Node.js event loop *not* exit so long as the
+timer is active. Calling `timer.ref()` multiple times will have no effect.
 
-If `callback` is not a function `setTimeout()` will throw immediately.
+Returns a reference to the timer object.
 
-## unref()
+### timer.unref()
 
-The opaque value returned by [`setTimeout`][] and [`setInterval`][] also has the
-method `timer.unref()` which allows the creation of a timer that is active but
-if it is the only item left in the event loop, it won't keep the program
-running. If the timer is already `unref`d calling `unref` again will have no
-effect.
+When called, allows the Node.js event loop to exit if the timer is the only
+item left in the Node.js event loop. Calling `timer.unref()` multiple times
+will have no effect.
 
-In the case of [`setTimeout`][], `unref` creates a separate timer that will
-wakeup the event loop, creating too many of these may adversely effect event
-loop performance -- use wisely.
+In the case of [`setTimeout`][], `timer.unref()` creates a separate timer that
+will wake the Node.js event loop. Creating too many of these can adversely
+impact the performance of the event.
 
-Returns the timer.
+Returns a reference to the timer object.
 
+[the Node.js Event Loop]: ../topics/the-event-loop-timers-and-nexttick.html
```

Comment:
Not yet. Will be double checking the links before landing.
On May 23, 2016 3:57 PM, "Anna Henningsen" <notifications@github.com> wrote:

> In doc/api/timers.md
> <https://github.com/nodejs/node/pull/6937#discussion_r64303631>:
>
> >
> > +[the Node.js Event Loop]: ../topics/the-event-loop-timers-and-nexttick.html
>
> Are you sure this works?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/nodejs/node/pull/6937/files/a8c72718f4500daa613cfe84416f19e8664f6378#r64303631>
>